OPENIDM-20798 End-User UI Removed from IDM Update README.md

stacywray-ping opened the pull request:
The standalone end-user UI has been removed from IDM 8.0. These updates do the following:
- Add a note to the top of the README that the end-user UI is no longer supported in PingIDM 8.0. Users must use the README for guidance and documentation.
- Added anchor refs for new items to Common Questions
- Moved customization options out of IDM docs into the README and formatted accordingly.

stacywray-ping committed:
OPENIDM-20798 End-User UI Removed from IDM Update README.md

The standalone end-user UI has been removed from IDM 8.0. These updates do the following:
- Add a note to the top of the README that the end-user UI is no longer supported in PingIDM 8.0. Users must use the README for guidance and documentation.
- Added anchor refs for new items to Common Questions
- Moved customization options out of IDM docs into the README and formatted accordingly.

Signed-off-by: stacywray-ping <stacywray@pingidentity.com>

stacywray-ping committed:
Editorial changes for consistency

- Added hyphens for End-User UI for consistency across page
- Fixed anchor refs for some Common Question titles

Signed-off-by: stacywray-ping <stacywray@pingidentity.com>

C-Masuda commented on README.md:
```diff
@@ -1,13 +1,16 @@
   <p align="center">
     Easy to integrate, standalone UI to demonstrate ForgeRock Identity Management.
     <br>
-    <a href="https://backstage.forgerock.com/docs/"><strong>Explore ForgeRock docs »</strong></a>
```
Should we link to the new Ping Docs?

stacywray-ping replied:
Yes, would make sense. Just link to 7.5 for now?

C-Masuda replied:
yeah, just set it and forget it. I imagine we'll have more edits to do here after the new platform UI is ready.

C-Masuda commented on README.md:
Should we remove or mark the section `How to Add a Self-Service Stage to the UI`? All the self-service stuff has been removed from IDM 8.0, so not sure that this applies. @jakefeasel @huck-elliott-forgerock ?

jakefeasel replied:
We should definitely make a note that this would only apply for 7.x, not 8.0

C-Masuda commented on README.md:
````diff
@@ -372,11 +384,37 @@ These instructions apply to registration, password reset, and forgotten username
+```
+"urlContextRoot" : "/exampleui",
+```
+Alternatively, to change the End-User UI URL in the admin UI, follow these steps:
````
Probably need to remove this alternative method wording and procedure, as self-service doesn't exist in the admin UI.

C-Masuda commented:
update all instance of `{secureHostname}` to match the rest of this readme: `localhost:8080`

C-Masuda commented on README.md:
```diff
@@ -372,11 +384,37 @@ These instructions apply to registration, password reset, and forgotten username
+The logout URL effectively performs the same action as clicking on the **Log out** link of the UI.
+
 <a name="how-to-add-additional-registration-flows"></a>
 ### How to Add Additional Registration Flows
```
This section also need to be marked with an admonition that it only applies to 7.5.x and earlier.

C-Masuda commented on README.md:
```diff
@@ -391,35 +429,126 @@ These instructions apply to registration, password reset, and forgotten username
 
 5. At the top of the `router/index.js` file, import the new Vue file and ensure that it matches the component that you specified in the route: `import RegistrationSecondFlow from '@/components/mains/RegistrationSecondFlow';`
-6. Make sure that your IDM access.js file is configured properly, based on [these docs](https://backstage.forgerock.com/docs/idm/6/integrators-guide/#uss-registration). You will see a forbidden access error if this file isn't configured correctly.
+6. Make sure that your IDM `access.js` file is configured properly, based on [these docs](https://backstage.forgerock.com/docs/idm/6/integrators-guide/#uss-registration). You will see a forbidden access error if this file isn't configured correctly.
```
Although this whole section is only for 7.5.x and earlier, let's still update this to a more modern link:
https://docs.pingidentity.com/pingidm/7.5/self-service-reference/uss-registration-config.html

C-Masuda commented on README.md:
```diff
@@ -1,13 +1,16 @@
 <p align="center">
   <b>Identity Management (End User) - UI</b>
+  <p align="center">
+    The End-User UI is no longer supported in PingIDM 8.0. Follow the documentation and guidance in this README.
```
Per @jakefeasel , let's downgrade this to:
> The End-User UI is not included in PingIDM 8.0 and later. Follow the documentation and guidance in this README.

C-Masuda commented on README.md:
````diff
@@ -391,35 +429,126 @@ These instructions apply to registration, password reset, and forgotten username
+
+The managed object record for end users who consent to sharing this data is shown in REST output and in the audit activity log as one `consentedMappings` object:
+
+```
````
Tip:
Most markdown allows you add highlighting if you specify the language after the first three backticks of your codeblock:
```markdown
```json
```

C-Masuda reviewed (commented):
Looks pretty good. Make sure there admonitions on all the self-service topics.

stacywray-ping committed:
Added edits Update README.md

- Added Curtis's and Jake's edits regarding "not included" vs "not supported"
- Added important admonitions on all self-service topics
- Added language to code markers
- Replaced {secureHost} with `local:8080`
- Updated 6.0 links to 7.5

Signed-off-by: stacywray-ping <stacywray@pingidentity.com>

andrewhertel commented on README.md:
The descriptions look accurate at this point. As Curtis mentions below, we will likely need to update this doc to some decent degree once we have the enduser within the Platform UI.

stacywray-ping committed:
Fixed End-User UI inconsistencies Update README.md

Fixed only one instance of capitalization inconsistencies with End-User UI.

Signed-off-by: stacywray-ping <stacywray@pingidentity.com>

stacywray-ping commented:
@andrewhertel Would you mind merging this PR since I have the necessary approvals? I don't have write access. Thanks in advance!
